runners/apex/src/main/java/org/apache/beam/runners/apex/ApexRunner.java

```diff
@@ -28,7 +28,8 @@
 import java.util.Arrays;
 import java.util.List;
 
-import org.apache.beam.runners.apex.translators.TranslationContext;
+import org.apache.beam.runners.apex.translation.ApexPipelineTranslator;
+import org.apache.beam.runners.apex.translation.TranslationContext;
```

Comment:
This import makes me think that maybe `ApexTranslator.translate(Pipeline, DAG)` would make sense, to make it so this class does not need to depend on the `TranslationContext`. It seems like an implementation detail leak.